bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/ScriptExtensionManagerWrapper.java

```diff
@@ -56,7 +57,7 @@ public Object get(String type) {
         return manager.getDefaultPresets();
     }
 
-    public void importPreset(String preset) {
-        manager.importPreset(preset, container.getFactory(), container.getScriptEngine(), container.getIdentifier());
+    public Map<String, Object> importPreset(String preset) {
```

Comment:
I know that you are not the initial contributor of these classes but may I ask you to add some JavaDoc to these methods?

Reply:
I've added docs to the this method. TBH I'm not so sure about the other methods as I haven't used them. I know that most of the actual implementation is in `ScriptExtensionManager`, which unfortunately is also missing docs.